src/Config/Source/CliSource.php

```diff
@@ -234,6 +235,12 @@ public function read(): Repository
             ]);
         }
 
+        if ($this->input->getOption(self::OPTION_WITHOUT_BLACKFIRE)) {
+            $repository->set([
+                self::PHP_DISABLED_EXTENSIONS => ['blackfire']
+            ]);
```

Comment:
@IgorVitol 
It looks like you are overwriting the value but you need to add
https://github.com/magento/magento-cloud-docker/blob/1ee887057e4fa5f5a6c30c8e204dbd1bdcd39463/src/Config/Source/CloudSource.php#L244

```suggestion
            $repository->push(self::PHP_DISABLED_EXTENSIONS, 'blackfire');
```